extern/region-4.9/configure

```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.63 for region 4.8.
+# Generated by GNU Autoconf 2.63 for region 4.7.
```

Comment:
Because that's the code I got from Mark. Most likely I changed that manually last time I incorporated changes from the `region` library, but I didn't do it this time.

There are a number of changes I have to make to these scripts because despite the efforts they still contain a number of dependencies on CIAO Makefiles, or mistakes like this. In principle, I should just be given a tar ball created with `make distcheck`, but in practice that's not the case.